send-to-bluetooth@asgharalitaj/send-to-bluetooth@asgharalitaj.nemo_action.in

```diff
@@ -0,0 +1,9 @@
+[Nemo Action]
+_Name=Send to bluetooth
+_Comment=Send to bluetooth: %F
```

Comment:
I would suggest send-via-bluetooth@AsgharAliTa as a proper name but it is ultimately your call.

Reply:
I am good with either.